Sources/Validation/Validators/PhoneValidator.swift

```diff
@@ -0,0 +1,90 @@
+
+public enum PhoneFormat {
```

Comment:
This enum and `PhoneType` both need doc blocks for them.

Reply:
Been waiting for this. Also consider having multiple country code formats. 
https://en.wikipedia.org/wiki/E.164